ratbagd/ratbagd-button.c

```diff
@@ -128,6 +128,8 @@ static int ratbagd_button_get_special(sd_bus *bus,
 	enum ratbag_button_action_special special;
 
 	special = ratbag_button_get_special(button->lib_button);
+	if (special == RATBAG_BUTTON_ACTION_SPECIAL_INVALID)
+		special = RATBAG_BUTTON_ACTION_SPECIAL_UNKNOWN;
```

Comment:
We should log when we fix up RATBAG_BUTTON_ACTION_SPECIAL_INVALID. It is likely a driver issue that should be fixed

Reply:
done!